lucene_schema.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
acl_schema.json?

can you make a similar one for CORD-19 with all the fields we have now